index.html

```diff
@@ -1,14 +1,34 @@
 <!doctype html>
 <html lang="en">
 <head>
+  <script src="0802.js"></script>
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>Hello World</title>
   <link href='/style.css' rel='stylesheet'>
 </head>
-<body>
-  <h1>Hello, world!</h1>
-  <button onclick="console.log('hello');">Press me</button>
-  <script defer type="text/javascript" src="/0802.js"></script>
+<body  class="main.page">
+<button onclick="test1()">Run test1</button>
+<button onclick="test2()">Run test2</button>
+<button onclick="test3()">Run test3</button>
```

Comment:
Make correct indentation. In VSCode -> https://code.visualstudio.com/docs/editor/codebasics#_formatting.
Later on, we will have projects with prettier and linter.

Always make the best code possible for review. Always.